### PR TITLE
test: Testing issue with data product tests on branch

### PR DESCRIPTION
**Pull request by energinet-ajw**
Populates the "wholesale_basis_data" data product/schema with views allowing for easier and modular access to the internal basis_data tables, based on calculation IDs. The first user of this product is the settlement reports.

Added basis data schemas and tests there of.

## Pull-request quality

- [x] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [x] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)

### CODE REVIEW

**Review by slb-halfspace — APPROVED** [review]
LGTM, though I am still unsure of how a missing contracts specification leads to other data product's tests failing.

File: source/databricks/calculation_engine/package/datamigration/migration_scripts/202409261100_create_view__wholesale_basis_data__charge_link_periods_v1.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
+DROP VIEW IF EXISTS {CATALOG_NAME}.{WHOLESALE_BASIS_DATA_DATABASE_NAME}.charge_link_periods_v1
```
Comment (JesperGomes) [inline source/databricks/calculation_engine/package/datamigration/migration_scripts/202409261100_create_view__wholesale_basis_data__charge_link_periods_v1.sql:1]: should we use CREATE OR REPLACE instead?

File: source/databricks/calculation_engine/contracts/data_products/wholesale_basis_data/charge_link_periods_v1.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import pyspark.sql.types as t
+
+nullable = True
+
+charge_link_periods_v1 = t.StructType(
+    [
+        # UUID
+        t.StructField("calculation_id", t.StringType(), not nullable),
+        #
+        # 'wholesale_fixing' | 'first_correction_settlement' |
```
Comment (JesperGomes) [inline source/databricks/calculation_engine/contracts/data_products/wholesale_basis_data/charge_link_periods_v1.py:10]: also 'aggregation'

Comment (JesperGomes) [reply source/databricks/calculation_engine/contracts/data_products/wholesale_basis_data/charge_link_periods_v1.py:10]: goes for the other views as well

**Review by JesperGomes — APPROVED** [review]

**Review by JesperGomes — APPROVED** [review]

**Pull request merged**